back/app/services/side_fx_cosponsorship_service.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class SideFxCosponsorshipService
+  include SideFxHelper
+
+  def after_accept(cosponsorship, user)
+    LogActivityJob.perform_later(
+      cosponsorship,
+      'input_cosponsorship_accepted',
```

Comment:
Detail, but why not go with `accepted` as the action name? The item and its type (`Cosponsorship`) is already part of the data.